documentation/src/main/asciidoc/user_guide/eviction.adoc

```diff
@@ -163,45 +165,57 @@ and if this limit is exceeded, some candidates are found to be removed according
+In clustered environments, nodes in the cluster can have different access times for the same entry. Entries do not expire from the cache until they reach the maxium idle time for all owners across the cluster.
 
-WARN: Clustered caches should always enable reaper when using max idle expiration due to some limitations listed in the section below.
+When a node detects that an entry has reached the maximum idle time and is expired, the node gets the last time that the entry was accessed from the other owners in the cluster. If the other owners indicate that the entry is expired, that entry is not returned to the user and removed from the cache.
+//dnaro: Returned to the user? Maybe returned to the requester?
 
-When using max idle, it is important to keep the following points in mind:
+The following points apply to using the `maxIdle` configuration with clustered caches:
 
-. A Cache.get in a clustered cache will return the "expired" entry if at least one owner finds that this entry is not expired. The last access time for the given entry will be updated to the current time.
-. When an entry is found to have expired due to max idle in the expiration reaper, instead of updating the last access time to current time, it will update all nodes to the most recent access of all owners.
-. Iteration in a clustered cache *will* return entries that this node thinks are expired via max idle. This is to ensure performance as no remote invocations are done. The entry is not refreshed and may be removed by the reaper or get access.
-. A clustered transactional cache does *NOT* remove entries expired via max idle on a get. These entries are only removed via reaper. This is prevented as doing so can cause deadlocking.
+* If one or more owner in the cluster detects that an entry is not expired, then a `Cache.get` operation returns the entry. The last access time for that entry is also updated to the current time.
+* When the expiration reaper finds entries that are expired with the maximum idle time, the reaper updates all nodes to the most recent access time instead of updating the last access time to the current time.
+//dnaro: Why would it update the last access time to the current time? Wouldn't that make the entry not expired?
+//dnaro: The reaper updates the expired entries across all nodes so that the access time for the entries is the most recent time?
+* Clustered transactional caches do *not* remove entries that are expired with the maximum idle time on get operations. These expired entries are removed with the expiration reaper thread only, otherwise deadlocking can occur.
+* Iteration across a clustered cache returns entries that might be expired with the maximum idle time. This behavior ensures performance because no remote invocations done during the iteration. However this does not refresh any expired entries, which are removed by the expiration reaper or when accessed directly (`Cache.get`).
```

Comment:
probably should be "because no remote invocations are performed during the iteration"